### PR TITLE
Addmodel

### PR DESCRIPTION
**Pull request by Pk05999**
remodify_model

### CODE REVIEW

File: model/employee.go [hunk 1]
```diff
@@ -1,4 +1,14 @@
...
 
+import uuid "github.com/google/uuid"
 
-type 
\ No newline at end of file
+type Employee struct {
+	ID         uuid.UUID `json:"id"`
+	Name       Name      `json:"name"`
+	Role       string    `json:"role"`
+	Degree     Degree    `json:"degree"`
```
Comment (Vishal2911) [inline model/employee.go]: Degrees []Degree

File: model/degree.go [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package model
+
+type Degree struct {
```
Comment (Vishal2911) [inline model/degree.go:3]: Fields should be like 
Degree Name , Type , Duration , start , end , percentage

File: model/facilities.go [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package model
+
+import "github.com/google/uuid"
+
+type Facilities struct {
+	ID     uuid.UUID `json:"id"`
+	Name   string    `json:"name"`
+	FType  string    `json:"f_type"`
```
Comment (Vishal2911) [inline model/facilities.go]: type

File: model/fee.go [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+import "github.com/google/uuid"
+
+type Fee struct {
+	ID     uuid.UUID `json:"id"`
+	Name   string    `json:"name"`
+	Type   Type      `json:"type"`
+	Amount int       `json:"amount"`
+}
+
+type Type struct {
```
Comment (Vishal2911) [inline model/fee.go]: For now comment this struct

File: model/fee.go [hunk 5]
```diff
@@ -0,0 +1,17 @@
+package model
+
+import "github.com/google/uuid"
+
+type Fee struct {
+	ID     uuid.UUID `json:"id"`
+	Name   string    `json:"name"`
+	Type   Type      `json:"type"`
```
Comment (Vishal2911) [inline model/fee.go]: Change Type to string

File: model/student.go [hunk 6]
```diff
@@ -0,0 +1,15 @@
+package model
+
+import uuid "github.com/google/uuid"
+
+type Student struct {
+	ID         uuid.UUID `json:"id"`
+	Name       Name      `json:"name"`
+	RollNumber int       `json:"roll_number"`
+	Address    Address   `json:"address"`
+	Contact_No Number    `json:"contact_no"`
```
Comment (Vishal2911) [inline model/student.go]: ContectDetails

File: model/subject.go [hunk 7]
```diff
@@ -0,0 +1,9 @@
+package model
+
+import "github.com/google/uuid"
+
+type Subject struct {
+	ID   uuid.UUID `json:"id"`
+	Name string    `json:"name"`
+	Book []Book    `json:"book"`
```
Comment (Vishal2911) [inline model/subject.go]: Books []Book

File: model/book.go [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+
+type Book struct {
+	ID        uuid.UUID `json:"id"`
+	BookName  string    `json:"book_name"`
+	Mrp       int64     `json:"mrp"`
+	Publisher string    `json:"publisher"`
+	Author    Author    `json:"author"`
+}
+
+type Author struct {
```
Comment (Vishal2911) [inline model/book.go:13]: Add address , Contact Details

File: model/class.go [hunk 9]
```diff
@@ -0,0 +1,12 @@
+package model
+
+import uuid "github.com/google/uuid"
+
+type Class struct {
+	ID           uuid.UUID `json:"id"`
+	RoomNo       int64     `json:"room_no"`
+	Subject      []Subject `json:"subject"`
```
Comment (Vishal2911) [inline model/class.go]: Subjects []Subject

File: model/class.go [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
+	Fees         Fee       `json:"fees"`
+	Teacher      []Teacher `json:"teacher"`
```
Comment (Vishal2911) [inline model/class.go]: Teachers []Teacher

File: model/class.go [hunk 11]
```diff
@@ -0,0 +1,12 @@
+package model
+
+import uuid "github.com/google/uuid"
+
+type Class struct {
+	ID           uuid.UUID `json:"id"`
+	RoomNo       int64     `json:"room_no"`
+	Subject      []Subject `json:"subject"`
+	Fees         Fee       `json:"fees"`
```
Comment (shivammpce) [inline model/class.go]: add fee in field not fees

File: model/common.go [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+	LastName  string `json:"last_name"`
+}
+
+type Address struct {
+	Line1  string `json:"line1"`
+	Line2  string `json:"line2"`
+	Nearby string `json:"near_by"`
+}
+
+type Number struct {
```
Comment (Vishal2911) [inline model/common.go:14]: Contact Details

File: model/event.go [hunk 13]
```diff
@@ -0,0 +1,15 @@
...
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type Event struct {
+	ID        uuid.UUID `json:"id"`
+	Name      string    `json:"name"`
+	Fees      Fee       `json:"fees"`
```
Comment (Vishal2911) [inline model/event.go:12]: fee

File: model/degree.go [hunk 14]
```diff
@@ -0,0 +1,10 @@
+package model
+
```
Comment (shivammpce) [inline model/degree.go:2]: change Degree field

File: model/student.go (hunk 6 above)
Comment (Vishal2911) [reply model/student.go]: No under score

**Review by Vishal2911 — APPROVED** [review]

**Review by shivammpce — APPROVED** [review]

**Pull request merged**